internal/llm/retry_test.go

```
@@ -0,0 +1,515 @@
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/genai"
+)
+
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)